ui/packages/diagnosis_report/src/utils/i18n.js

```diff
@@ -0,0 +1,35 @@
+import i18next from 'i18next'
```

Comment:
We should avoid duplicating this utilities

Reply:
yep, I agree, don't know how to do yet.

Reply:
Can we extract i18n to a standalone package?

Reply:
yep, that's a way, we can do it later.